app/models/response/emcsTfe/GetMovementResponse.scala

```diff
@@ -66,5 +68,56 @@ case class GetMovementResponse(
 }
 
 object GetMovementResponse {
-  implicit val reads: Reads[GetMovementResponse] = Json.reads
+
+implicit lazy val reads: Reads[GetMovementResponse] = for {
+  arc <- (__ \ "arc").read[String]
+  sequenceNumber <- (__ \ "sequenceNumber").read[Int]
+  destinationType <- (__ \ "destinationType").read[DestinationType]
+  consignorTrader <- (__ \ "consignorTrader").read[TraderModel]
+  consigneeTrader <- (__ \ "consigneeTrader").readNullable[TraderModel]
+  deliveryPlaceTrader <- (__ \ "deliveryPlaceTrader").readNullable[TraderModel]
+  placeOfDispatchTrader <- (__ \ "placeOfDispatchTrader").readNullable[TraderModel]
+  firstTransporterTrader <- (__ \ "firstTransporterTrader").readNullable[TransportTraderModel]
+  deliveryPlaceCustomsOfficeReferenceNumber <- (__ \ "deliveryPlaceCustomsOfficeReferenceNumber").readNullable[String]
+  localReferenceNumber <- (__ \ "localReferenceNumber").read[String]
+  eadStatus <- (__ \ "eadStatus").read[String]
+  dateOfDispatch <- (__ \ "dateOfDispatch").read[LocalDate]
+  journeyTime <- (__ \ "journeyTime").read[String]
+  documentCertificate <- (__ \ "documentCertificate").readNullable[Seq[DocumentCertificateModel]]
+  eadEsad <- (__ \ "eadEsad").read[EadEsadModel]
+  headerEadEsad <- (__ \ "headerEadEsad").read[HeaderEadEsadModel]
+  transportMode <- (__ \ "transportMode").read[TransportModeModel]
+  movementGuarantee <- (__ \ "movementGuarantee").read[MovementGuaranteeModel]
+  transportDetails <- (__ \ "transportDetails").read[Seq[TransportDetailsModel]]
+  numberOfItems <- (__ \ "numberOfItems").read[Int]
+  reportOfReceipt <- (__ \ "reportOfReceipt").readNullable[ReportOfReceiptModel]
+  items <- (__ \ "items").read[Seq[MovementItem]]
+  eventHistorySummary <- (__ \ "eventHistorySummary").readNullable[GetMovementHistoryEventsResponse]
+} yield {
+  GetMovementResponse(
+    arc = arc,
+    sequenceNumber = sequenceNumber,
+    destinationType = destinationType,
+    consignorTrader = consignorTrader,
+    consigneeTrader = consigneeTrader,
+    deliveryPlaceTrader = deliveryPlaceTrader,
+    placeOfDispatchTrader = placeOfDispatchTrader,
+    firstTransporterTrader = firstTransporterTrader,
+    deliveryPlaceCustomsOfficeReferenceNumber = deliveryPlaceCustomsOfficeReferenceNumber,
+    localReferenceNumber = localReferenceNumber,
+    eadStatus = eadStatus,
+    dateOfDispatch = dateOfDispatch,
+    journeyTime = journeyTime,
+    documentCertificate = documentCertificate,
+    eadEsad = eadEsad,
+    headerEadEsad = headerEadEsad,
+    transportMode = transportMode,
+    movementGuarantee = movementGuarantee,
+    transportDetails = transportDetails,
+    numberOfItems = numberOfItems,
+    reportOfReceipt = reportOfReceipt,
+    items = items,
+    eventHistorySummary = eventHistorySummary
+  )
+}
```

Comment:
Could you fix the indentation of this code, please? IntelliJ should be able to sort it out